docs/topics/mulled.rst

```diff
@@ -92,6 +92,49 @@ you could do something along these lines.
   
    $ mulled-build push 'pandoc=1.17.2--0' --test 'pandoc --help' -n biocontainers
 
+Build Singularity containers from Docker containers
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Singularity containers can be built from Docker containers using the mulled-update-singularity-containers command.
+
+To generate a single container:
+
+.. code-block:: bash
+
+   $ mulled-update-singularity-containers --containers samtools:1.6--0 --logfile /tmp/sing/test.log --filepath /tmp/sing/ --installation /usr/local/bin/singularity
+
+.. code-block:: bash
+
+``--containers`` indicates the container name (here ``samtools:1.6--0``), ``--logfile`` the name of a log file containing test results, ``--filepath`` the location where the containers should be placed, and ``--installation`` the location of the Singularity installation. This can be found using ``whereis singularity``.
+
+Multiple containers can be installed simultaneously by giving ``--containers`` more than one argument:
+
+.. code-block:: bash
+
+   $ mulled-update-singularity-containers --containers samtools:1.6--0 --logfile /tmp/sing/test.log --filepath /tmp/sing/ --installation /usr/local/bin/singularity
+
+.. code-block:: bash
+
+For a large number of containers, it may be more convenient to employ the ``--container-list`` option:
+
+.. code-block:: bash
+
+   $ mulled-update-singularity-containers --container-list list.txt --logfile /tmp/sing/test.log --filepath /tmp/sing/ --installation /usr/local/bin/singularity
+
+.. code-block:: bash
+
+Here ``list.txt`` should contain a list of containers, each on a new line.
+
+Alternatively, invoking the ``--all`` argument will build all Docker containers located at https://quay.io/organization/biocontainers/ which do not have a corresponding Singularity container at https://depot.galaxyproject.org/singularity/.
```

Comment:
which will build Singularity Containres for all Docker images located at ....
This tool will internally do the same what ... 1) create list 2) create singularity containers ... will do ...